### DISCUSSION TITLE
Shortcuts to Sound files

### DISCUSSION THREAD

**Discussion by rydfree** [EdgeTX/edgetx#3081] [Ideas]
I searched but could not find if there had been a suggestion to allow a jump within the list when selecting a sound file.  When you want a special function to play a track it would be nice if at the top of the list there were A-Z so that you could just click on the letter to carry you directly to that section . It sure would save a lot of wear and tear on the scroll wheel.

**Comment by pfeerick** [comment]
All 26 letters would probably be overkill/unwieldy to use, but A-D, E-H, I-L, etc navigation steps would be good. The only discussion I am aware of on this topic so far was that "something" would be nice... but no more concrete an idea. So thank you for starting that off! :) Perhaps using the existing toolbar sidebar used on the sources and switches menus, and then you can also use the page keys to jump between the groups of letters, or restrict the list to just those letters...

**Reply by rydfree** [reply]
Yes the A-D , E-H, format would work well .  Could probably keep it simple and even do a A-I , J-R , S-Z .  3 choices would be an improvement .

**Comment by Airfiddler** [comment]
It would help speed the search for a track if the scroll wheel would work in both directions. If I'm looking for a track
that starts with "W" currently I can only scroll foreword. Scrolling backward would get me to "W" a heck of a lot faster.
At 72 I don't have a lot of time left, so any speedup in finding what I'm looking for will help.   :=}
